docs/data_structures.md

```diff
@@ -0,0 +1,74 @@
+# MapSwipe 2.0 Data Model
+The MapSwipe crowdsourcing workflow is designed following an approach already presented by [Albuquerque et al. (2016)](http://www.mdpi.com/2072-4292/8/10/859). Four concepts are important in the following: projects, groups, tasks and results.
+
+
+<img src="/img/mapswipe_data_model.png">
+
+
+## Projects
+A project in MapSwipe is primarily characterized by an area of interest, a set of satellite imagery tiles and a feature type to look for. In addition, each project defines the number of users that are requested to classify each individual satellite imagery tile. Furthermore, background information on the context of the mapping request is provided. The information is uploaded through a web form by project managers.
+
+| Parameter | Description | Possible Values | 
+| --- | --- | --- |
+| **Type** | What type of work is required from the user| 1 = FIND_BUILT_AREAS (the original tasks) <br/> 2 = VALIDATE_BUILDING_FOOTPRINTS |
+| **Id** |	Each project has a unique identifier. This Id will be generated automatically and cannot be chosen by the project manager. |
+| **Name** | Each project has a name, which will be shown in the app. Often project in the same region have similar names, e.g. “Madagascar 1” and “Madagascar 2”. |
+| **Geometry** | A project is characterized by its multi polygon geometry. Thus, projects can theoretically consist of several distinct regions. Nevertheless, most projects focus on a single area. |
+| **Redundancy** | Project managers can define how often each individual task will be classified by MapSwipe volunteers at minimum. For most projects this redundancy is set to three. |
+| **Imagery Provider** | This parameter refers to a provider of a tile map service. For projects of this study imagery is provided by Bing. Tiles of satellite imagery can be obtained from a tile map service endpoint. Each tile can be identified using a quad key representation of its x, y and z coordinates. Tiles are projected in WGS 84 Web Mercator (Auxiliary Sphere). This corresponds to the EPSG code 3857.
+| **Project Details** | The project details describe the goal and scope of the project. This is visualized in the app and is important to stir the volunteer’s motivations. In general, the project is described by five to ten sentences. |
+| **Look For** | The parameter defines which type of features are mapped in the project. This will be visualized in the mapping interface of the app. For this study, volunteers are always asked to look for buildings only. |
+
```

Comment:
additionally we could add "group size" as an attribute to each project. This time group size would refer to the number of tasks in a group. Adding it here, would allow us to have different sizes for different projects of the same type.

Reply:
I think that's a really good idea. Agreed ;)

Reply:
```
{
  "contributors" : 0,
  "groupAverage" : 0.0,
  "id" : "1002",
  "image" : "http://www.redcrosseth.org/media/k2/items/cache/5a05a447acfdf6fcc40548cc4c1cea8d_L.jpg",
  "importKey" : "-LK7IpMFNJ7mwe2IQych",
  "isFeatured" : false,
  "lookFor" : "BUILDINGS",
  "name" : "Map vulnerable communities in Ethiopia 4",
  "progress" : 0,
  "projectDetails" : "Ethiopia has a population of over 102 million, yet many people live in remote and rural locations. These communities are vulnerable to flooding, droughts and other extreme climate related events. The Red Cross, with the support of the IKEA Foundation is asking you to help map and locate these rural communities, allowing the Red Cross help prepare them for any pending disasters. Please swipe slowly through the satellite imagery and tap anything that looks like it could be a building or village.",
  "state" : 3,
  "tileserver" : "bing",
  "type" : 1,
  "verificationCount" : "3",
  "zoom" : 18
}
```